tests/generic_views/test_edit.py

```diff
@@ -281,8 +289,14 @@ def test_update_with_interpolated_redirect(self):
                         {'name': 'Randall Munroe (author of xkcd)', 'slug': 'randall-munroe'})
         self.assertQuerysetEqual(Author.objects.all(), ['<Author: Randall Munroe (author of xkcd)>'])
         self.assertEqual(res.status_code, 302)
-        pk = Author.objects.all()[0].pk
+        pk = Author.objects.first().pk
         self.assertRedirects(res, 'http://testserver/edit/author/%d/update/' % pk)
+        # Also test with escaped chars in URL
+        res = self.client.post('/edit/author/%d/update/interpolate_redirect_nonascii/' % a.pk,
```

Comment:
Could you wrap it according to PEP8. I know this style is heavily used in this file, but we should change that eventually.

```python
res = self.client.post(
    '/edit/author/%d/update/interpolate_redirect_nonascii/' % a.pk,
    {'name': 'John Doe', 'slug': 'john-doe'}
)